custom-namespaces.yaml

```diff
@@ -854,6 +854,10 @@ firefox_desktop:
       tables:
         - events_table_view: urlbar_events_table
           base_table: mozdata.firefox_desktop.urlbar_events
+    urlbar_events_daily_table:
```

Comment:
You can also have an explore be generated automatically on top of this table view if you'd like. Here's an example:
https://github.com/mozilla/lookml-generator/blob/main/custom-namespaces.yaml#L673-L689